csv_file: "/home/sergiosierra/catkin_ws/src/control_ubosque/config/paths/path_points1.csv"
csv_header: True
delay: 5
frame_id: "odom"
path_topic: "/path"
control_rate: 10
update_params_service: "/update_parameters"